app/src/features/FhirAttributePanel/Condition.tsx

```diff
@@ -143,48 +147,57 @@ const Condition = ({ condition }: ConditionProps): JSX.Element => {
     }
 
     if (column.id && column.table && column.column && column.owner) {
-      const newColumn = column.pending
-        ? await createColumn({
-            columnRequest: column as ColumnRequest,
-          }).unwrap()
-        : await updateColumn({
-            id: column.id,
-            columnRequest: column as ColumnRequest,
-          }).unwrap();
-
-      dispatch(
-        columnUpdated({
-          id: column.id,
-          changes: { ...newColumn, pending: false },
-        })
-      );
+      try {
+        const newColumn = column.pending
+          ? await createColumn({
+              columnRequest: column as ColumnRequest,
+            }).unwrap()
+          : await updateColumn({
+              id: column.id,
+              columnRequest: column as ColumnRequest,
+            }).unwrap();
 
-      if (condition.id && newColumn.id !== condition.column) {
         dispatch(
-          conditionUpdated({
-            id: condition.id,
-            changes: { column: newColumn.id },
+          columnUpdated({
+            id: column.id,
+            changes: { ...newColumn, pending: false },
           })
         );
-      }
+        if (condition.id && newColumn.id !== condition.column) {
+          dispatch(
+            conditionUpdated({
+              id: condition.id,
+              changes: { column: newColumn.id },
+            })
+          );
+        }
+      } catch (error) {}
     }
   };
   const handleDeleteCondition = async () => {
-    !condition.pending &&
-      condition.id &&
-      (await deleteCondition({ id: condition.id }));
-    condition.id && dispatch(conditionRemoved(condition.id));
+    try {
+      !condition.pending &&
+        condition.id &&
+        (await deleteCondition({ id: condition.id }));
+      condition.id && dispatch(conditionRemoved(condition.id));
+    } catch (error) {}
   };
   const handleRelationChange = (
     event: React.ChangeEvent<{ name?: string | undefined; value: unknown }>
   ) => {
-    if (condition.id)
+    if (condition.id) {
+      const value = event.target.value as ConditionRelationEnum;
```

Comment:
naming is a bit confusing: I suggest 
```suggestion
      const relation = event.target.value as ConditionRelationEnum;
```